images.js

```diff
@@ -8,9 +8,11 @@ module.exports = {
     "https://i.ytimg.com/vi/iLtEcfqbBco/maxresdefault.jpg", // xi pooh
     "https://cdn.dribbble.com/users/415975/screenshots/1740996/attachments/281613/Mona_Lisa_Bear_dribBig.jpg",
     "https://i.imgur.com/FAtw00Q.jpg", // bear vs bull
-    "https://makeavideo.studio/assets/A_confused_grizzly_bear_in_calculus_class.__21.webp"
+    "https://makeavideo.studio/assets/A_confused_grizzly_bear_in_calculus_class.__21.webp",
```

Comment:
will this show up in Webex correctly?

Reply:
file type is not jpeg/gif like the other ones

Reply:
Line 11 already exists, so yes?